packages/blade/src/components/Tabs/tabTokens.ts

```diff
@@ -0,0 +1,174 @@
+import type { TabsProps } from './types';
+import type { DotNotationSpacingStringToken } from '~utils/types';
+
+type Devices = 'desktop' | 'mobile';
```

Comment:
can we use this type from `useBreakpoint`?

Reply:
We have a `DeviceType` type defined there, I've exported it.

Reply:
done.